vortex-array/src/array/varbinview/builder.rs

```diff
@@ -1,119 +0,0 @@
-use std::marker::PhantomData;
-use std::mem;
-use std::mem::ManuallyDrop;
-
-use arrow_buffer::NullBufferBuilder;
-use vortex_dtype::DType;
-
-use crate::array::primitive::PrimitiveArray;
-use crate::array::varbinview::{BinaryView, Inlined, Ref, VarBinViewArray, VIEW_SIZE};
-use crate::validity::Validity;
-use crate::{ArrayData, IntoArray, IntoArrayData, ToArray};
-
-pub struct VarBinViewBuilder<T: AsRef<[u8]>> {
-    views: Vec<BinaryView>,
-    nulls: NullBufferBuilder,
-    completed: Vec<ArrayData>,
-    in_progress: Vec<u8>,
-    block_size: u32,
-    phantom: PhantomData<T>,
-}
-
-impl<T: AsRef<[u8]>> VarBinViewBuilder<T> {
-    pub fn with_capacity(capacity: usize) -> Self {
-        Self {
-            views: Vec::with_capacity(capacity),
-            nulls: NullBufferBuilder::new(capacity),
-            completed: Vec::new(),
-            in_progress: Vec::new(),
-            block_size: 16 * 1024,
```

Comment:
we have a diff default block size (16K) vs arrow (8K), which is a slight behavior difference